<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteControl</class>
 <widget class="QMainWindow" name="RouteControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>1061</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="init_btn">
         <property name="text">
          <string>Выбрать видео для детекции</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="carClassification">
         <property name="text">
          <string>Выбрать машину для распознования</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ok">
         <property name="text">
          <string>Продолжить видео</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Плотность потока: 3 ед. на км</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Интенсивность потока: 3 ед/мин</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Средняя скорость: 20 км/ч</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="road_view">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
